### PR TITLE
Hw4

### PR DESCRIPTION
**Pull request by thenandback**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  <body>
+    <header id="site-header">
+      <div id="main-title">Austin's Portfolio</div>
+      <nav id="main-nav">
+        <ul>
+            <li class="tab" data-content="home"><a href="#">Home</a></li>
+            <li class="spacer"><a href="#">|</a></li>
+            <li class="tab" data-content="about"><a href="#">About</a></li>
+            <li class="spacer"><a href="#">|</a></li>
+            <li class="tab" data-content="articles"><a href="#">Blog</a></li>
```
Comment (RyuGuardian) [inline index.html]: You've got some extra spacing at the beginnings of lines 17-21.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        </header>
+          <div class="blogColumn" id="myImg">
+            <img src="images/myImage.jpg" />
+          </div>
+          <div class="blogColumn" id="myFacts">
+            <div class="fact"></div>
+            <div class="fact"></div>
+            <div class="fact"></div>
+            <div class="fact"></div>
+          </div>
```
Comment (RyuGuardian) [inline index.html]: The <div>s from lines 53-61 should be lined up with their <header> sibling above them.

File: scripts/article.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  return template(this);
+}
+
+blogData.forEach(function(ele) {
+  blogList.push(new Article(ele));
+});
+
+// for large windows, adds blogList to shortest column
+//for short windows, alternates adding columns (since we can't compare column lengths)
+blogList.forEach(function(element, index){
```
Comment (RyuGuardian) [inline scripts/article.js]: I'm wary of this function. I feel like all this could be taken care of in CSS using media queries.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    <hr id="mainHr">
+
+    <main>
+
+      <section id="articles" class="tab-content">
+        <div id="blogIntro"><img src="http://www.homesteadstencilco.com/fpdb/images/6620.jpeg" /></div>
+        <div id="leftColumn" class="blogColumn"></div>
+        <div id="rightColumn" class="blogColumn"></div>
+
+        <script id="article-template" type="text/x-handlebars-template">
```
Comment (sjschmidt44) [inline index.html]: Typically your Handlebars templates will go up in the head, and then you'll target a destination element further down in the body. It helps identify the templates you're using up top, in a similar way to how we declare general variables at the top of our script tags.

**Pull request merged**